src/SwedbankPay/Core/Library/Methods/Card.php

```diff
@@ -74,6 +75,12 @@ public function initiateCreditCardPayment($orderId, $generateToken, $paymentToke
             ]
         ];
 
+        if ($this->configuration->getData(ConfigurationInterface::CHECKOUT_METHOD) ===
+            ConfigurationInterface::METHOD_SEAMLESS
+        ) {
+            $params['payment']['urls']['paymentUrl'] = $urls->getPaymentUrl();
```

Comment:
What does `$urls->getPaymentUrl()` return? Where is the method defined?

Reply:
@asbjornu It returns the checkout page where js code should be placed. It defined there https://github.com/SwedbankPay/swedbank-pay-woocommerce-core/pull/46/commits/ef0957debba787b66ec2d224e4fad5d0cfdfa143#diff-e4e3b2fe15e4bfb33e862bc52da037baaae2287f76484e46d97f422d0bb83bf4R13
and there
https://github.com/SwedbankPay/swedbank-pay-woocommerce-core/pull/46/commits/ef0957debba787b66ec2d224e4fad5d0cfdfa143#diff-9c236e1e67d9a25d7c8e78ccbe515ead23ecfa910e08d4910db7f16739a79ca7R15